client/my-sites/checklist/checklist-show/index.jsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ *
+ * @format
+ */
+
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+
+import Card from 'components/card';
+import Checklist from 'components/checklist';
+import Main from 'components/main';
+import DocumentHead from 'components/data/document-head';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { getSiteChecklist } from 'state/selectors';
+import QuerySiteChecklist from 'components/data/query-site-checklist';
+
+class ChecklistShow extends PureComponent {
+	render() {
+		const { siteId, translate } = this.props;
+
+		return (
+			<Main>
+				<DocumentHead title={ translate( 'Site Checklist' ) } />
+				{ siteId && <QuerySiteChecklist siteId={ siteId } /> }
+				<Card>
+					<h1 className="checklist-show__header-heading">{ translate( 'Welcome back!' ) }</h1>
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Welcome Back :)' )` *ES Score: 15*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/checklist-component-to-checklist-route)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).